contents/handbook/strategy/roadmap.md

```diff
@@ -7,32 +7,21 @@ showTitle: true
 Our mission is to increase the number of successful products in the world. 
 
 Our roadmap for 2021 will do three things:
-1. Create a solid core product that's easy to use
-2. Ensure the best developer platform for event-based analytics
-3. Set PostHog up to service huge volumes
+1. Reach parity with other product analytics tools, but better
+2. Ensure we have the best developer platform for event-based analytics
+3. Make it easy to deploy and maintain a scalable version of PostHog, for free
 
-# 1. Core product
+# 1. Reach parity
 
-PostHog is a product that people love, primarily because it covers 90% of analytics use cases but bundled into one package.
-Some examples of the functionality we've built last year:
+PostHog went from 0 to a fully-featured analytics platform in an incredibly short period of time. Realistically, that did mean we had to cut some corners when it came to stability, usability and depth of analytics you could do with PostHog.
 
-- Product analytics
-- Session recording
-- Feature flags
-- Heatmaps
-- Autocapture
+In 2021, we're going to make sure that PostHog has all the features you love from other platforms. That way, self-hosting your product analytics doesn't need to be a compromise.
```

Comment:
Should we specify what "other platforms" we mean?

Ie:

- product analytics and feature flags?